modules/fargate/monitoring/lambda_service_utilization_monitoring.tf

```diff
@@ -0,0 +1,93 @@
+data "aws_iam_policy_document" "lambda_service_utilization_monitoring" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "ecs:*",
+      "cloudwatch:*",
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = [
+      "arn:aws:logs:*:*:*",
+    ]
+  }
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "elasticloadbalancing:*",
+    ]
+
+    resources = [
+      "*",
+    ]
+  }
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "cloudwatch:*",
+      "ecs:*",
+    ]
+
+    resources = [
+      "*",
+    ]
+  }
+}
+
+data "archive_file" "lambda_service_utilization_monitoring-dotfiles" {
+  type        = "zip"
+  output_path = "${path.module}/lambda_service_utilization_monitoring.zip"
```

Comment:
It seems like this `.zip` is being checked in with the code. I suggest adding it `.gitignore`.

Reply:
In this case, the zip archive is generated as it should be (via `archive_file` resource). In case of dashboard lambda function, it is written in python and requires Jinja2 module. Unless I simplify it not to use that module (which is possible with a little bit amount of time), we would have to struggle with this issue https://github.com/telia-oss/concourse-tasks/issues/29#issuecomment-564017823 all over again. Other possibility would be to upload the zip archive into some public bucket and fetch it from there.

Reply:
Zip files are gone now :-)